Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.mm

```diff
@@ -578,14 +578,15 @@ static bool hasValue(const ImageBufferBackendHandle& backendHandle)
 
     layer.contentsOpaque = m_isOpaque;
 #if HAVE(SUPPORT_HDR_DISPLAY_APIS)
-    layer.contentsHeadroom = m_maxRequestedEDRHeadroom;
     ALLOW_DEPRECATED_DECLARATIONS_BEGIN
     if (layer.wantsExtendedDynamicRangeContent) {
         // Painted contents have already been tonemapped, so disable it on the layer.
         if (isDelegatedDisplay)
             layer.toneMapMode = CAToneMapModeIfSupported;
```

Comment:
Should you set layer.contentsHeadroom to 0 in this case? (Or will it never switch back&forth?)